templates/index.twig

```diff
@@ -3,13 +3,28 @@
 {% block content %}
 <div class="col-md-12">
 	{% include "article-list.twig" with {"wordCount": 100} %}
-	<div class="pagination-buttons">
+<div class="text-center">
+	<ul class="pagination pagination-centered">
 		{% if pagination.prev %}
-		<a href="{{pagination.prev.link}}" class="prev {{pagination.prev.link|length ? '' : 'invisible'}} pull-left"><button class="btn btn-default">Previous</button></a>
+			<li>
+					<a class="btn" href="{{pagination.prev.link}}" class="prev {{pagination.prev.link|length ? '' : 'invisible'}}">Previous</a>
+			</li>
 		{% endif %}
+		{% for page in pagination.pages %}
+				<li>
+						{% if page.link %}
+								<li><a href="{{page.link}}" class="btn">{{page.title}}</a></li>
```

Comment:
The `btn` class needs to be removed from here too, as it's giving the links a rounded corner, which we don't want.